cvc5_z3py_compat/z3.py

```diff
@@ -3057,6 +3057,149 @@ def Gt(a, b):
     return a > b
 
 
+#########################################
+#
+# Transcendentals
+#
+#########################################
+
+
+def Pi(ctx=None):
+    """ Create the constant pi
+
+    >>> Pi()
+    Pi
+    """
+    ctx = get_ctx(ctx)
+    return _to_expr_ref(ctx.solver.mkTerm(Kind.Pi, []), ctx)
```

Comment:
```suggestion
    return _to_expr_ref(ctx.solver.mkTerm(Kind.Pi), ctx)
```
This should not be necessary